tests/test_plugins.py

```diff
@@ -39,63 +16,26 @@
     "outputs": None,
     "flags": [],
     "icon": None,
-    # "env": "conda create -n test-env python=3.6.7",
-    # "requirements": "pip: numpy",
+    "env": "",
+    "requirements": "",
     "dependencies": [],
 }
 
 
-@pytest.fixture(name="engine")
-async def setup_engine(event_loop):
-    """Set up engine."""
-    logger = _LOGGER
-    opt = parse_cmd_line(["--debug", "--token", TOKEN])
-    opt = prep_env(opt, logger)
-    opt = bootstrap(opt, logger)
-    engine = Engine(opt, logger)
-    engine.setup()
-    await engine.async_start()
-    print(engine)
-    yield engine
-    await engine.async_stop()
-
-
-@pytest.fixture(name="client")
-async def mock_client(engine, event_loop):
-    """Provide a mock client."""
-    client_id = str(uuid.uuid4())
-    session_id = str(uuid.uuid4())
-    client = FakeClient(URL, client_id, session_id, TOKEN, event_loop)
-    await client.connect()
-    await client.register_client()
-    yield client
-
-
-async def test_debugging(engine):
-    """Try to figure out what is going on."""
-    print(engine)
-    # print(client)
-    assert True
-    assert False
-
-
 @pytest.fixture(name="init_plugin")
```

Comment:
Perhaps rename it to test_plugin? Init_plugin sounds like a function name, here it's more like a plugin object.